### ISSUE TITLE
Worried about project development pace

### ISSUE THREAD

**Issue by getub** [twitter/typeahead.js#1320]
The project's development has slowed to a crawl over the past 11 months. There was a brief hope of revival with 0.11, but it's silent again. There were lots of issues accumulated over a year that were rapidly closed with the hope that 1.0 would fix a lot of pending issues. 

It seems that there's no clear day-to-day ownership of the project. I'm wondering how does the community feel about the pace of development. Also curious if there's any interest in forking over the project and giving it a new lease of life.

I don't mean to belittle Jake's wonderful work, but he has been busy and there are lots of live projects that depend on typeahead. It seems fair to discuss the roadmap and figure out if it can be revived or should one look for alternatives. 

Any comments on alternatives : [select2](https://github.com/select2/select2), [chosen](https://github.com/harvesthq/chosen) and [selectize.js](https://github.com/brianreavis/selectize.js/)

**Comment by Ghostavio** [comment]
I have to +1 this, it's a little bit sad to see such an amazing project with so many pending issues.

**Comment by robvon** [comment]
select2 looks promising. It's active and has just about everything I want

**Comment by stefanoTron** [comment]
+1; I have been waiting for a few bug fixes for MONTHS... :(

**Comment by fkammer** [comment]
+1

I really love typeahead.js as it is a very flexible and powerful solution. I also like the clean style of this project, especially the separation of the actual search engine as I'm using it a lot as a replacement of search filters in Angular.js for more complex data structures. But like @getub I'm a bit concerned about the long list of issues and the long waiting Pull Requests.

@jharding started this project and I really appreciate all of his work. Thank you Jake! I was also happy when I saw that this project moved to the twitter repository as I thought that Twitter would support this by giving Jake more time to focus on typeahead and maybe even with some manpower. Unfortunately this seems not to be the case.

I think there are a lot of people in the community that are willing to help out, fixing some issues or implementing new features like fuzzy search for bloodhound (me!).
But as far as I know there is no public roadmap which is a big problem in my eyes. What is planed for v1.0 and what for the next minor releases? It makes it hard to start working on an issue that came up a couple of times when you don't know if it will be merged and for which version it is planed and when you see the long list of pending PRs.

Don't get me wrong, I don't want to cut Jake off! But I would appreciate if you could publish a roadmap or work one out with the community and if this project could get one or two additional maintainers. As @getub mentioned a lot of projects depend on typeahead so let's get it back to life!

**Comment by ronakshah94** [comment]
+1 
I agree with @fkammer . If there were a simple roadmap laid out for the contributors to work on, that would've immensely helped. I came across this project a few weeks ago, and I am trying to get on board with it. I want to contribute to the development of the project, but I was concerned with the number of issues and pull requests that were piling up! But it's great to see that there are people who want to keep this repository active!

**Comment by jlbooker** [comment]
I offered (shortly before v0.11 was released) to help keep the GitHub issues triaged and categorized. I'm still happy to help out in that manner to get things moving again.

**Comment by ronakshah94** [comment]
@jlbooker If you could do that, that would be great!

**Comment by getub** [comment]
What's a good way to find the more actively used branch that should be forked ? Is it `0.10.5` or `0.11.01`.  I feel that latter was an attempt to revive but its not well tested and used yet. Whereas the former has been around for more than a year and should have more interest.

**Comment by gaborbernat** [comment]
version 0.11.01 is a good one I think, using it in production, just needs some fixes and better testing :) also many of the pull requests we could use are for 0.11.01

**Comment by Ghostavio** [comment]
Starting from next week I can also help on anything necessary to keep this going. I don't have much experience on keeping opensource projects but I'm happy to give it a try with everyone else who expressed their will to help on this thread.

**Comment by lenovouser** [comment]
@getub I'd be willing to help with starting a new fork. But my JavaScript knowledge isn't that good yet. So I only would be helpful with maintaining the project's pull requests and issues or the project as a whole.

**Comment by yahya-uddin** [comment]
+1000 on new contributors. I would be happy to help (especially on the Bloodhound side)!

**Comment by Ghostavio** [comment]
should we do it?

**Comment by lenovouser** [comment]
Yes. I just created the **[CoreJS](https://github.com/corejavascript)** organization since I think there is someone who has to take the initiative. I invited:
- @getub
- @Ghostavio
- @jlbooker
- @anik786

and created the [corejavascript/typeahead.js](https://github.com/corejavascript/typeahead.js) repository. I did that because I believe we should start clean, pull down the master from here and push it up to the new organization. Also because lot's of people requested features like being able to use it without jQuery. We can of course also direct fork it, but I would prefer my variant. What do you guys think?

**Comment by yahya-uddin** [comment]
I think it would be much better for a direct fork, due to the simple fact Twitter guys may comeback. Although then again looking at the alternatives, I think I may just work on those!

**Comment by Ghostavio** [comment]
I agree with @anik786

**Comment by ahmetabdi** [comment]
@ApfelUser Direct fork IMO, are we go?

**Comment by lenovouser** [comment]
Yes, sorry for the late response:

[corejavascript/typeahead.js](https://github.com/corejavascript/typeahead.js)

**Comment by getub** [comment]
Great job, looking forward to an active community.

**Comment by ustun** [comment]
I think a good option is to contact Algolia and ask them to develop this plugin. They are a search company and they seem to be suggesting this library in their docs (see https://www.algolia.com/doc/tutorials/multi-auto-complete), so I believe they could help.

**Comment by macedd** [comment]
For those willing to use the latest patches:

`bower install -S "https://github.com/corejavascript/typeahead.js.git#master"`

Great work guys..!

**Comment by Ghostavio** [comment]
awesome!

**Comment by redox** [comment]
> I think a good option is to contact Algolia and ask them to develop this plugin. They are a search company and they seem to be suggesting this library in their docs (see https://www.algolia.com/doc/tutorials/multi-auto-complete), so I believe they could help.

Actually that's what we did :) Here is Algolia's [autocomplete.js fork](https://github.com/algolia/autocomplete.js). It's based on typeahead.js 0.10.5 on which we've added a few features (custom menu template, built-in sources, angular support, removed jquery, ...).
